systest/scripts/install_test_infra.sh

```diff
@@ -60,5 +55,9 @@ git clone\
   ${NEUTRON_LBAAS_REPO} \
   ${NEUTRON_LBAAS_DIR}
 
+# create directories for copying tempest.conf file
+
+mkdir ${TEMPEST_VENV_ETC_DIR}
```

Comment:
You can use `mkdir -p` here which will make the full path if it doesn't exist and not fail if it does exist. You can then not have the mkdir mkdir